src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -195,81 +195,86 @@ comment_id | Line/noticeAssignments/NoticeAssignment/NoticeRef | This field is p
+direction_type | Route/DirectionType | The value of this field is used without transformation.
 destination_id |  | The `DirectionRef` of the Route doesn't link to a stop (neither stop_point nor stop_area), thus its value is not used.
-line_id | *Route/LineRef/@ref* | This field is prefixed. 
 
 **ServiceJourneyPattern references**
-All ServiceJourneyPattern of a `route` are stored as complementary `object_codes`. ServiceJourneyPattern nodes are listed in the same parent node as Route nodes.
+
+All ServiceJourneyPattern of a `route` are stored as complementary `object_codes`. ServiceJourneyPattern nodes are listed in the nodes `CompositeFrame/frames/GeneralFrame[]/members/ServiceJourneyPattern[]`. The `GeneralFrame` to be used is the one with the `@ref` containing `NETEX_STRUCTURE`.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
 object_type |  | fixed value `route` 
-object_id | *ServiceJourneyPattern/RouteRef/@ref* | This field is prefixed. 
+object_id | ServiceJourneyPattern/RouteRef/@ref | This field is prefixed. 
 object_system |  | fixed value `Netex_ServiceJourneyPattern`
-object_code | *ServiceJourneyPattern/@id* | The value of this field is used without transformation.
+object_code | ServiceJourneyPattern/@id | The value of this field is used without transformation.
 
 
 ### trips.txt
-`trips` are described in each **offre_** file in the nodes **ServiceJourney**. 
+`trips` are described in each **offre_*.xml** file in the nodes **CompositeFrame/frames/GeneralFrame[]/members/ServiceJourney[]**. 
+The `GeneralFrame` to be used is the one with the `@ref` containing `NETEX_STRUCTURE`.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
-route_id | *ServiceJourney/JourneyPatternRef/@ref* | route_id of the Route containing the JourneyPatternRef as an object_code.
+route_id | ServiceJourney/JourneyPatternRef/@ref | route_id of the Route containing the `JourneyPatternRef` as an object_code.
 service_id |  | Defined using `ServiceJourney/DayTypeRef`, see (calendar.txt and calendar_dates.txt)[]
-trip_id | *ServiceJourney/@id* | This field is prefixed. 
-trip_headsign | *ServiceJourney/DestinationDisplayRef* | Content of the `DestinationDisplay/FrontText` node. If not available, the name of the `stop_point` of the last `stop_time` is used.
-trip_short_name | *ServiceJourney/DestinationDisplayRef* | Content of the `DestinationDisplay/PublicCode` node. If not available, this field is empty.
-company_id | *ServiceJourney/OperatorRef* | if *ServiceJourney/OperatorRef* is not defined, use *Line/OperatorRef* in [*lines.xml*](#linestxt) file. This field is prefixed.
-physical_mode_id | *Line/TransportMode* | see physical_modes definition
+trip_id | ServiceJourney/@id | This field is prefixed. 
+trip_headsign | ServiceJourney/DestinationDisplayRef | Content of the `DestinationDisplay/FrontText` node. If not available, the name of the `stop_point` of the last `stop_time` is used.
```

Comment:
In the data we have (which might be incomplete), I can see some of those `DestinationDisplay` node but they are in the `NETEX_STRUCTURE` frame, while we are parsing trips in the `NETEX_HORAIRE` frame. It might worth mentioning this if I'm correct.

On top of that, this line contains an indirection: we shouldn't use `ServiceJourney/DestinationDisplayRef` as a value but as a reference to jump somewhere else in the XML to get the value. There is the same problem below with `trip_short_name` et `company_id`. I don't know if would want to express that notion in a better way?

Reply:
Nice catch for the first one, `DestinationDisplay` are in the `ServiceJourneyPattern` object (french spec was OK :-/ ). I've tried a clarification for the double use of ServiceJourneyPattern and ServiceJourney, hope it will be good :)

For the references, i did the best I could and I find the current specs understandable. As I try not to be too verbose, let's keep it like that if it's ok with you.